pkgs/servers/monitoring/buildkite-agent-metrics/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, go, buildGoPackage, fetchFromGitHub }:
+
+let
+  goPackagePath = "github.com/buildkite/buildkite-agent-metrics";
+in
+buildGoPackage rec {
+  pname = "buildkite-agent-metrics";
+  version = "5.1.0";
+  goDeps = ./deps.nix;
+
+  inherit goPackagePath;
+
+  preBuild = ''
+    rm -rf go/src/${goPackagePath}/vendor
+  '';
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "buildkite";
+    repo = "buildkite-agent-metrics";
+    sha256 = "1dx8vhp07yc5c02la7iq5mxxj2lglsm4jzfb5vsac0vn70c9dwci";
+  };
+
+  buildFlagsArray = let
+    t = "${goPackagePath}/vendor/github.com/buildkite/buildkite-agent-metrics/version";
+  in ''
+    -ldflags=
+       -X ${t}.Version=${version}
+       -X ${t}.Revision=unknown
+       -X ${t}.Branch=unknown
+       -X ${t}.BuildUser=nix@nixpkgs
+       -X ${t}.BuildDate=unknown
+       -X ${t}.GoVersion=${lib.getVersion go}
+  '';
+
+  doCheck = true;
+
+  meta = with lib; {
+    description = "A command-line tool (and Lambda) for collecting Buildkite agent metrics";
+    homepage = "https://github.com/buildkite/buildkite-agent-metrics";
+    license = licenses.mit;
+    maintainers = with maintainers; [ ];
```

Comment:
Are you not willing to be a maintainer for this software? there is a push to try and reduce the number of unmaintained packages in nixpkgs

Reply:
Hi @alexarice, thanks for the reviews.

I am happy to be maintainer but ofBorg failed when I put my name in it, so a second commit removed my name so I could see the tests pass.

Reply:
Have you got an entry in `maintainers.nix`?